libs/SmartSync/src/com/salesforce/androidsdk/smartsync/target/SyncDownTarget.java

```diff
@@ -91,6 +94,12 @@ public SyncDownTarget() {
         super();
     }
 
+
+    public SyncDownTarget(String idFieldName, String modificationDateFieldName) {
```

Comment:
Previously id field name and modification date field name could be specified when building from json but not with other constructors